src/components/PDOKAutoSuggest/PDOKAutoSuggest.test.tsx

```diff
@@ -43,6 +43,28 @@ const renderAndSearch = async (value = 'Dam', props = {}) => {
   await screen.findByTestId('auto-suggest')
 }
 
+const renderAndSearchWithStreetNameOnlyAsTrue = async (
+  value = 'dor',
+  props = {}
+) => {
+  render(
+    withAppContext(
+      <PDOKAutoSuggest onSelect={onSelect} streetNameOnly={true} {...props} />
```

Comment:
you can leave {true}

Reply:
why not add a third arg to renderAndSearch with streetNameOnly?

Reply:
Euh, isn't streetNameOnly by default false and that's why I set it here expressly to true?